src/main/drivers/timer_hal.c

```diff
@@ -1050,3 +1050,22 @@ HAL_StatusTypeDef DMA_SetCurrDataCounter(TIM_HandleTypeDef *htim, uint32_t Chann
     return HAL_OK;
 }
 
+int8_t timerGetNumber(const TIM_TypeDef *tim)
+{
+#define TIM_COUNT(tim, base) (((unsigned)tim - base) >> 10)
```

Comment:
This is a bit dangerous - lookupTimerIndex uses this trick in case, it will work is distance is bigger (sacrificing some performance) and will fail if timer spacing is smaller - you get working code or compile error.

Reply:
This code is gone now.